assets/elements/environment/crab/crab.atlas.yaml

```
@@ -1,4 +1,4 @@
 image: ./crab.png
-tile_size: [17, 12]
+tile_size: [17, 13]
```

This should actually be:

```suggestion
tile_size: [18, 12]
```

---

Hi @zicklag, thanks for the feedback. 

Ok so I gave it another shot. 

I updated the following: 

* crab.atlas.yaml to match your suggestions
* crab.png + sproinger.png to match your requirements as well. 

Hopefully this works, but again if changes need to be made please let me know.